### PR TITLE
[Fixes #1841] Change XML DCS and XmlSerializer output formatters to not ...

### LINKED ISSUE #1841: Change XML DCS and XmlSerializer output formatters to not derive from the base XmlOutputFormatter

### PR DESCRIPTION
**Pull request by kichalla**
...derive from the base XmlOutputFormatter

@yishaigalatzer @harshgMSFT

### CODE REVIEW

**Comment by rynowak** [comment]
Please change all the unit tests to use the default constructor for the formatter (since that constructor creates a default settingts), then :shipit: from me.

@harshgMSFT or @sornaks should have a look also.

File: src/Microsoft.AspNet.Mvc.Core/Formatters/XmlSerializerOutputFormatter.cs [hunk 1]
```diff
@@ -58,6 +91,17 @@ protected virtual XmlSerializer CreateSerializer([NotNull] Type type)
             }
         }
 
+        /// <summary>
+        /// Creates a new instance of <see cref="XmlWriter"/> using the given stream and the WriterSettings.
```
Comment (harshgMSFT) [inline src/Microsoft.AspNet.Mvc.Core/Formatters/XmlSerializerOutputFormatter.cs]: cref for WriterSettings

File: test/Microsoft.AspNet.Mvc.Core.Test/Formatters/XmlDataContractSerializerOutputFormatterTests.cs [hunk 2]
```diff
@@ -85,12 +85,50 @@ public static IEnumerable<object[]> BasicTypeValues
...
+            Assert.False(writerSettings.CloseOutput);
+            Assert.False(writerSettings.CheckCharacters);
+        }
+
+        [Fact]
+        public async Task SuppliedWriterSettings_TakeAffect()
+        {
+            // Arrange
+            var writerSettings = FormattingUtilities.GetDefaultXmlWriterSettings();
+            writerSettings.OmitXmlDeclaration = false;
```
Comment (harshgMSFT) [inline test/Microsoft.AspNet.Mvc.Core.Test/Formatters/XmlDataContractSerializerOutputFormatterTests.cs]: assuming this is true by default.

Comment (kichalla) [inline test/Microsoft.AspNet.Mvc.Core.Test/Formatters/XmlDataContractSerializerOutputFormatterTests.cs]: yes its true by default...the above test verifies this..

**Comment by harshgMSFT** [comment]
:shipit: from me once you fix minor cref comment.

**Comment by kichalla** [comment]
Commit: 0e9091f0eb7ba7db0334b04d14e47163b14d726d

**Pull request closed without merging**